browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -131,6 +139,14 @@ bool BraveTabContextMenuContents::IsBraveCommandIdEnabled(
     case BraveTabMenuModel::CommandShowVerticalTabs:
     case BraveTabMenuModel::CommandUseFloatingVerticalTabStrip:
       return true;
+    case BraveTabMenuModel::CommandToggleTabMuted: {
+      auto* model = static_cast<BraveTabStripModel*>(controller_->model());
+      for (const auto& index : model->GetTabIndicesForCommandAt(tab_index_)) {
+        if (!model->GetWebContentsAt(index)->GetLastCommittedURL().is_empty())
```

Comment:
I think `if (model->GetWebConentsAt(index)->IsCurrentlyAudible())` is better. WDYT?

Reply:
Hmmm the thing is I really want to show it above the `Mute site` entry, which is shown regardless of whether a site is currently playing audio.

We have a bunch of issues where people think `Mute site` means `Mute tab` and we want to start implying what the difference is. `Mute site` is super confusing :D

Reply:
Yeah, it's quite confusing :)
What I understand is `Mute site` is not related whether current tab is audible or is playing.
If user muted site, context menu has `Unmute iste` even it's not audible and not played.
It looks like mute site is cached property for that specific url.
Instead, I think `Mute tab` entry seems to be synced with tab's speaker icon status.

Reply:
I'll double check with @rebron to confirm :smile:

Reply:
We're going to keep the menu in :smile: